README.md

```diff
@@ -660,6 +661,38 @@ These are values that represent the current state of the downshift component.
 As a convenience, the `id` and `itemToString` props which you pass to
 `<Downshift />` are available here as well.
 
+## EventHandlers
+
+You can provide your own event handlers to Downshift which will be called before the default handlers.
+
+If you would like to prevent the default handler behavior in some cases, you can set the event's `preventDownshiftDefault` property to `false`:
+
+```javascript
+<input
+  {...getInputProps({
+    onKeyDown(event) {
```

Comment:
Let's change this to an arrow function just in case someone new copy/pastes this and tries to use `this` and doesn't understand why it's not working.